lib/fresh_connection/version.rb

```diff
@@ -1,4 +1,4 @@
 module FreshConnection
-  VERSION = "2.3.0"
+  VERSION = "2.3.1"
```

Comment:
Since it can not be merged like this, please restore it at the end.
If you want to change the version for test installation, use RC 1 etc.

```ruby
VERSION = "2.3.0.rc1"
```